<ui version="4.0" >
 <class>runIndicatorBatch</class>
 <widget class="QDialog" name="runIndicatorBatch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Run indicator batch</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="co_start_year" />
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="co_end_year" />
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Every</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="co_every_year" />
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>years</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>runIndicatorBatch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>runIndicatorBatch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
